doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -611,7 +611,8 @@ The code for integrating MCUboot into |NCS| is located in the :file:`ncs/nrf/mod
 
 The following list summarizes both the main changes inherited from upstream MCUboot and the main changes applied to the |NCS| specific additions:
 
-|no_changes_yet_note|
+
+* Fixed an issues related to referencing arm-vector table of the application which causes jumping to wrong address instead of the application reset vector for some builds when zephyr LTO (linking time optimization) was enabled.
```

Comment:
```suggestion
* Fixed an issue related to referencing the arm-vector table of the application, which causes jumping to wrong address instead of the application reset vector for some builds when zephyr LTO (Link Time Optimization) was enabled.
```

Reply:
https://developer.arm.com/documentation/den0042/a/Exceptions-and-Interrupts/Exception-priorities/The-Vector-table